src/website/app/demos/Table/examples/sortable.js

```diff
@@ -40,9 +46,9 @@ Dairy column sorts by length of the string rather than alphabetically.
   scope: { Table, sharedData },
   source: `
     () => {
-      const sortByLength = (a, b, column) => {
-        const lengthA =a[column].length;
-        const lengthB =b[column].length;
+      const sortByLength = (a, b, key) => {
+        const lengthA = a[key].length;
+        const lengthB = b[key].length;
         return lengthA < lengthB ? -1 : lengthA > lengthB ? 1 : 0;
```

Comment:
Since this functionality is used in multiple places, would it be better to implement the exported const here & add expose it in a separate comment in this example?